src/main/groovy/com/readytalk/gradle/js/JsPlugin.groovy

```diff
@@ -14,9 +14,9 @@ import org.gradle.api.plugins.BasePlugin
 import org.gradle.api.plugins.ExtraPropertiesExtension
 
 class JsPlugin implements Plugin<Project> {
-  static final String DEFAULT_NODE_VERSION = '0.12.7'
+  static final String DEFAULT_NODE_VERSION = '4.2.3'
```

Comment:
Personally, I'd rather default to the latest stable (5.2), rather than the latest LTS (4.2.3), but the bump is appreciated either way.